src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -50,26 +50,30 @@ interface AstroAdapter {
 	exports?: string[];
 	args?: any;
 	adapterFeatures?: AstroAdapterFeatures;
-	supportedAstroFeatures?: AstroFeatureMap;
+	supportedAstroFeatures: AstroAdapterFeatureMap;
```

Comment:
This was [renamed](https://github.com/withastro/astro/blob/948ad8eecf71f1f6de33855a6065925d7858d28d/packages/astro/src/types/public/integrations.ts#L100).